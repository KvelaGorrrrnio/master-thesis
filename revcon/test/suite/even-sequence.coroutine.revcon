@coroutine

coroutine main(list int seq) {
  local int i
  local int tmp
  from i == 0
  do
    if i == 2 or i == 4 or i == 6 or i == 8 or i == 10 then
      tmp += i
      push tmp seq
    else skip
    fi i == 2 or i == 4 or i == 6 or i == 8 or i == 10
  loop i += 1
  until i == 10
  delocal int tmp
  delocal int i 10
}
