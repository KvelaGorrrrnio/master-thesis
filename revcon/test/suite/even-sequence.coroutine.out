test/suite/even-sequence.coroutine.revcon:
[store]
  main: Ã˜
  seq: [10, 8, 6, 4, 2]

[stack]
[spawn main main begin () | resume main (seq) | despawn main main end ()] <|> []

